.github/workflows/ci.yml

```diff
@@ -35,3 +35,6 @@ jobs:
           yarn prepublish
       - name: test
         run: yarn ci
+      - name: Run spectron
+        id: spectron
+        uses: IAmMorrow/run-headless@v4
```

Comment:
(can we later move it to LedgerHQ? would need maybe a more obvious name, like `ledger-live-github-action-headless` or something)

Reply:
If it works, you can move it 
https://github.com/LedgerHQ/actions

Reply:
yep and we'll take the command to run as input